Sources/Request/Request/RequestParams/AnyParam.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  
+//
+//  Created by brennobemoura on 15/11/20.
+//
+
+import Foundation
+
+/// A way to create a custom `RequestParam`
+/// - Important: You will most likely want to use one of the builtin `RequestParam`s, such as: `Url`, `Method`, `Header`, `Query`, or `Body`.
+public struct AnyParam: RequestParam {
```

Comment:
Please mark this as deprecated, since they can now conform to `RequestParam` themselves:
```swift
@available(*, deprecated, message: "`AnyParam` is deprecated. Please conform to the `RequestParam` protocol instead.")
```